### PR TITLE
TeSS Search Snippet

### PR DESCRIPTION
**Pull request by christopherchild**
- This will allow a link through to the TeSS search page given a specific search term.
- Styling based on the existing class: registry_logo
- Example `{% include tess.html search="Research Management Plan" %}`

Issue: #145

### CODE REVIEW

File: _includes/tess.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<a class="no_icon nohover" data-toggle="tooltip" title="TeSS Search: {{include.search}}" href="https://tess.elixir-europe.org/search?q={{include.search}}">
+  <img style="background-color: orange;" class="registry_logo" src="images/tess-logo.png" alt="TeSS logo"/>
```
Comment (bedroesb) [inline _includes/tess.html:2]:
```suggestion
  <img style="background-color: #f47d20;" class="registry_logo" src="images/tess-logo.png" alt="TeSS logo"/>
```

Comment (bedroesb) [reply _includes/tess.html:2]: I would differentiate here from the color from bio.tools. I took the same color from the website  of tess

**Comment by bedroesb** [comment]
Working nicely!
Screenshot from 2020-10-16 11-25-59

I was just wondering, do we want to add a little text like on my screenshot by default before the button?
@pinarpink @christopherchild

**Comment by christopherchild** [comment]
> I was just wondering, do we want to add a little text like on my screenshot by default before the button?

That makes sense to me. The button gets a little lost otherwise.

**Comment by bedroesb** [comment]
I think this can go out of draft no :) ?

**Comment by christopherchild** [comment]
> I think this can go out of draft no :) ?

Unless you want me to update with some text like your screenshot?

**Comment by bedroesb** [comment]
maybe not a bad idea. Don't know if my example sentence is a good one ...

Or this is also possible:

Screenshot from 2020-10-16 11-42-55

```html
<a class="no_icon nohover" data-toggle="tooltip" title="TeSS Search: {{include.search}}" href="https://tess.elixir-europe.org/search?q={{include.search}}">
  Topic related trainings: <img style="background-color: #f47d20;" class="registry_logo" src="images/tess-logo.png" alt="TeSS logo"/>
</a>

```

**Comment by martin-nc** [comment]
*Grammar police alert * 'Training' is an uncountable noun, so it can't have a plural. You can have 'training courses' and 'training materials' but not 'trainings'. Could we change it to 'training', do you think?

**Comment by christopherchild** [comment]
> *Grammar police alert * 'Training' is an uncountable noun, so it can't have a plural. You can have 'training courses' and 'training materials' but not 'trainings'. Could we change it to 'training', do you think?

I will, don't worry.

**Comment by bedroesb** [comment]
Without a doubt @martin-nc thanks for the info!

**Comment by bedroesb** [comment]
One last thing: 

I like to use svg's for logos as much as possible because it scales beautiful. I made an svg for the logo of tess

[tess_logo.zip](https://github.com/elixir-europe/rdm-toolkit/files/5390728/tess_logo.zip)

@martin-nc do you think this is overkill? I did it with all the other logo's on the website too

**Comment by christopherchild** [comment]
> One last thing:
> 
> I like to use svg's for logos as much as possible because it scales beautiful. I made an svg for the logo of tess
> 
> [tess_logo.zip](https://github.com/elixir-europe/rdm-toolkit/files/5390728/tess_logo.zip)
> 
> @martin-nc do you think this is overkill? I did it with all the other logo's on the website too

I'll update it.

**Comment by martin-nc** [comment]
@bedroesb makes sense to me. We'll probably need a larger logo somewhere else in the site so an SVG would be good. As long as the TeSS folk are happy it's their official logo!

**Review by bedroesb — APPROVED** [review]

**Pull request merged**